test/e2e/certificates_test.go

```diff
@@ -68,10 +68,11 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 	BeforeEach(func() {
 		By("waiting for operator status to become available")
 		err := verifyOperatorStatusCondition(certmanageroperatorclient,
-			[]string{certManagerControllerDeploymentControllerName,
-				certManagerWebhookDeploymentControllerName,
-				certManagerCAInjectorDeploymentControllerName},
-			validOperatorStatusConditions)
+			GenerateConditionMatchers(
+				[]string{certManagerController,
+					certManagerWebhook,
+					certManagerCAInjector},
+				validOperatorStatusConditions))
```

Comment:
Would it be helpful to log the unmatched condition status before exiting? In that case, we may identify the underlying problem more easily without having to inspect the pod logs first.

Reply:
Yes, I'm about to follow-up where we can dump the conditions when status condition checks failed.